apps/codeforafrica/src/payload/blocks/PostsOverView.js

```diff
@@ -0,0 +1,66 @@
+import linkGroup from "../fields/links/linkGroup";
+
+const primaryTags = [
+  {
+    label: "Stories",
+    value: "stories",
+  },
+  {
+    label: "Opportunities",
+    value: "opportunities",
+  },
+];
+
+const PostsOverview = {
+  slug: "posts-overview",
+  imageURL: "/images/cms/blocks/posts_overview.jpg",
+  imageAltText: "Stories & Opportunities Overview",
+  fields: [
+    {
+      name: "title",
+      label: "Title",
+      required: true,
+      type: "text",
+    },
+    {
+      name: "primaryTag",
+      label: "Post Type",
+      required: true,
+      type: "select",
+      options: primaryTags,
+      hasMany: false,
```

Comment:
I think this block is only for stories... we don't have a featured view for opportunities in the designs.